### PR TITLE
cert_management_automation: mod_md Apache module

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: cert_management_automation/README.md [hunk 1]
```diff
@@ -67,6 +67,22 @@ with solver being either:
...
+[2](https://www.server-world.info/en/note?os=Fedora_35&p=httpd&f=10),
+[3](https://frasertweedale.github.io/blog-redhat/posts/2020-05-07-ipa-acme-mod_md.html),
+[4 (czech only, this is where I found it)](https://www.root.cz/clanky/https-certifikat-let-s-encrypt-pomoci-apache-a-vestaveneho-modulu-mod-md)
+
+The thing is that we haven't configured httpd directly since
+[we started using mod_wsgi-express](https://github.com/packit/packit-service/pull/1363)
+which configures httpd for us.
+
+Do we need to request [mod_wsgi](https://modwsgi.readthedocs.io)
+to support mod_md in a similar way it supports mod_ssl?
```
Comment (TomasTomecek) [inline cert_management_automation/README.md:84]: couldn't hurt :D I believe many other users would benefit from it

**Review by TomasTomecek — COMMENTED** [review]
mod_wsgi-express still creates a httpd.conf:

/tmp/mod_wsgi-localhost:8000:1000/httpd.conf

maybe we could call `mod_wsgi-express-3 setup-server`, let it create the config, and then append the mod_md configuration and call start afterwards

**Comment by csomh** [comment]
Was cert management automation even used before switching to mod_wsgi-express? IIRC, I've removed `mod_md` in packit/packit-service#1363, b/c I couldn't find any use of it.

**Comment by jpopelka** [comment]
> Was cert management automation even used before switching to mod_wsgi-express?

no, it wasn't AFAIK

> IIRC, I've removed `mod_md` in [packit/packit-service#1363](https://github.com/packit/packit-service/pull/1363), b/c I couldn't find any use of it.

wow, didn't know we had been installing it

**Review by TomasTomecek — APPROVED** [review]

**Pull request merged**